### PR TITLE
Building Variety 2

### PR DESCRIPTION
**Pull request by FarkasKBence**
#### Summary
SUMMARY: Content "Adds even more layouts to buildings "
#### Purpose of change
Created more variations to buildings and slightly changed ones that already exist.
#### Describe the solution
Some buildings have only 1 layout, so I made 1 more.
#### Screenshots

Bar
bar

Dojo

dojo

Gym

gym

Laundromat

laundromat

Pizza Parlor

pizza_parlor

Fast Food Restaurant

restaurant_fast

Teashop

teashop

### CODE REVIEW

**Comment by chaisawlajatang** [comment]
Oh god, that looks amazing.

File: data/json/mapgen/bar.json [hunk 1]
```diff
@@ -212,6 +212,99 @@
         { "item": "bar_trash", "x": 22, "y": [ 17, 18 ], "chance": 70, "repeat": [ 4, 5 ] },
         { "item": "bar_bathroom", "x": [ 2, 4 ], "y": [ 21, 22 ], "chance": 30, "repeat": [ 1, 2 ] }
       ]
+    }    
+    },
```
Comment (ZhilkinSerg) [inline data/json/mapgen/bar.json]:
```suggestion
},
```

File: data/json/mapgen/bar.json [hunk 2]
```diff
@@ -212,6 +212,99 @@
...
+    {
```
Comment (ZhilkinSerg) [inline data/json/mapgen/bar.json]:
```suggestion
{
```

File: data/json/mapgen/bar.json [hunk 3]
```diff
@@ -212,6 +212,99 @@
...
+        "|--BBBB--|dd___Dlllll{| ",
+        "  |-''-|__dd___|&&&ll{| ",
+        "       F_______|------| ",
+        "       f_______|        ",
+        "       FFFFFFFFF        "
+      ],
+      "terrain": {
+        "_": "t_pavement",
+        "~": "t_sidewalk",
+        " ": ["t_grass", "t_grass", "t_dirt", "t_shrub", "t_grass", "t_dirt" ],
```
Comment (ZhilkinSerg) [inline data/json/mapgen/bar.json]:
```suggestion
        " ": [ "t_grass", "t_grass", "t_dirt", "t_shrub", "t_grass", "t_dirt" ],
```

File: data/json/mapgen/bar.json [hunk 4]
```diff
@@ -212,6 +212,99 @@
...
+        "u": "t_floor",
+        "@": "t_console_broken",
+        "D": "t_door_locked",
+        "c": "t_linoleum_gray",
+        "d": "t_pavement",
+        "O": "t_rock_floor",
+        "F": "t_chainfence",
+        "f": "t_chaingate_c",
+        "l": "t_linoleum_gray",
+		"i": "t_linoleum_gray",
```
Comment (ZhilkinSerg) [inline data/json/mapgen/bar.json]:
```suggestion
        "i": "t_linoleum_gray",
```

File: data/json/mapgen/gym.json [hunk 5]
```diff
@@ -79,7 +79,7 @@
             "+": "t_door_c",
             ",": "t_sidewalk",
             "-": "t_wall",
-            ".": "t_grass",
+            ".": ["t_grass", "t_grass", "t_grass", "t_grass", "t_grass", "t_shrub", "t_dirt"],
```
Comment (ZhilkinSerg) [inline data/json/mapgen/gym.json]:
```suggestion
            ".": [ "t_grass", "t_grass", "t_grass", "t_grass", "t_grass", "t_shrub", "t_dirt" ],
```

File: data/json/mapgen/gym.json [hunk 6]
```diff
@@ -104,5 +104,94 @@
...
+            "~": "t_water_pool",
+            "{": "t_floor",
+            "!": "t_floor",
+            "#": "t_floor",
+            "H": "t_floor",
+            ":": "t_linoleum_white",
+            "*": "t_linoleum_white",
+            "+": "t_door_c",
+            ",": "t_sidewalk",
+            ".":  ["t_grass", "t_grass", "t_grass", "t_grass", "t_grass", "t_shrub", "t_dirt"],
```
Comment (ZhilkinSerg) [inline data/json/mapgen/gym.json]:
```suggestion
            ".":  [ "t_grass", "t_grass", "t_grass", "t_grass", "t_grass", "t_shrub", "t_dirt" ],
```

**Comment by AMurkin** [comment]
Good job.

Note for further PRs: [Guidelines](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/.github/CONTRIBUTING.md#guidelines).

File: data/json/mapgen/bar.json [hunk 7]
```diff
@@ -213,5 +213,98 @@
         { "item": "bar_bathroom", "x": [ 2, 4 ], "y": [ 21, 22 ], "chance": 30, "repeat": [ 1, 2 ] }
       ]
     }
+},
```
Comment (ZhilkinSerg) [inline data/json/mapgen/bar.json]:
```suggestion
  },
```

File: data/json/mapgen/bar.json [hunk 8]
```diff
@@ -213,5 +213,98 @@
...
+{
```
Comment (ZhilkinSerg) [inline data/json/mapgen/bar.json]:
```suggestion
  {
```

File: data/json/mapgen/gym.json [hunk 9]
```diff
@@ -104,5 +104,94 @@
...
+            "~": "t_water_pool",
+            "{": "t_floor",
+            "!": "t_floor",
+            "#": "t_floor",
+            "H": "t_floor",
+            ":": "t_linoleum_white",
+            "*": "t_linoleum_white",
+            "+": "t_door_c",
+            ",": "t_sidewalk",
+            ".":  [ "t_grass", "t_grass", "t_grass", "t_grass", "t_grass", "t_shrub", "t_dirt" ],
```
Comment (ZhilkinSerg) [inline data/json/mapgen/gym.json]:
```suggestion
            ".": [ "t_grass", "t_grass", "t_grass", "t_grass", "t_grass", "t_shrub", "t_dirt" ],
```

File: data/json/mapgen/gym.json [hunk 10]
```diff
@@ -104,5 +104,94 @@
...
+            "*": "f_shower",
+            "@": "f_treadmill",
+            "O": "f_locker",
+            "V": "f_exercise",
+            "C": "f_bench",
+            "c": "f_bench",
+            "T": "f_toilet",
+            "t": "f_sink"
+          },
+            "place_monsters": [ { "monster": "GROUP_MAPGEN_POOL", "x": [ 12, 19 ], "y": [ 18, 21 ] } ],
```
Comment (ZhilkinSerg) [inline data/json/mapgen/gym.json]:
```suggestion
          "place_monsters": [ { "monster": "GROUP_MAPGEN_POOL", "x": [ 12, 19 ], "y": [ 18, 21 ] } ],
```

File: data/json/mapgen/gym.json [hunk 11]
```diff
@@ -104,5 +104,94 @@
...
+            "toilets": { "T": {} },
```
Comment (ZhilkinSerg) [inline data/json/mapgen/gym.json]:
```suggestion
          "toilets": { "T": {  } },
```

File: data/json/mapgen/gym.json [hunk 12]
```diff
@@ -104,5 +104,94 @@
...
+            "items": {
```
Comment (ZhilkinSerg) [inline data/json/mapgen/gym.json]: Items node still formatted improperly, should be just one line:

```JSON
          "items": { "O": { "item": "gym", "chance": 80 }, "H": { "item": "vending_drink", "chance": 75, "repeat": [ 4, 8 ] } }
```

**Comment by Pupsi-Mupsi** [comment]
well done!

**Pull request merged**